src/components/Security/Users/Page.vue

```diff
@@ -106,3 +104,17 @@ export default {
   }
 }
 </script>
+
+<style lang="scss" scoped>
+::v-deep .usersDropdown {
```

Comment:
wow nice :) 
I didn't know this `v-deep` tricks

Reply:
This is something @berthieresteban did and I'm not sure how it works either

Reply:
v-deep is just a way to unscoped the following class
Just like if we add another `<style>` block without scoped and add the `usersDropdown` class in  it